common/src/main/java/org/opensearch/ml/common/model/MLGuard.java

```diff
@@ -159,4 +159,9 @@ public Boolean validateStopWordsSingleIndex(String input, String indexName, List
         }
         return hitStopWords.get();
     }
+
+    public enum Type {
```

Comment:
May be we can add a from method like [this](https://github.com/opensearch-project/ml-commons/blob/main/common/src/main/java/org/opensearch/ml/common/FunctionName.java#L34)

Let's rename it to `MLGuardType`?

Reply:
It's an inner class which is used as MLGuard.Type. So I don't think we need to add MLGuard prefix in the name.

Reply:
can this MLGuard.Type be used in the Guardrails type also? OR these two types are different?